src/app/blocks/graphql-block/graphql-block.component.ts

```diff
@@ -3,6 +3,7 @@ import {BaseBlockComponent} from '../base-block/base-block.component';
 import {HttpClient} from '@angular/common/http';
 import { get, isObject } from 'lodash';
 import {mappingUtility} from '../mapping-block/mapping-util';
+import { SharedStateService } from 'src/app/services/shared-state.service';
```

Comment:
GraphQl gets a small update to allow it to access state. This makes it possible to access shared git config data.